From fa1b3798477b91492330a6db9e81834d64ce8ec1 Mon Sep 17 00:00:00 2001
From: "Andreas K. Huettel" <mail@akhuettel.de>
Date: Thu, 17 Jan 2013 17:33:46 +0100
Subject: [PATCH] Revert "Use a slightly better strategy for symlinking:
 although the previous"

This reverts commit 4e8ee28f3a314edef2a8489854c0ed58ac668070.
---
 tools/pykdeuic4/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/pykdeuic4/CMakeLists.txt b/tools/pykdeuic4/CMakeLists.txt
index 092985d..2209fba 100644
--- a/tools/pykdeuic4/CMakeLists.txt
+++ b/tools/pykdeuic4/CMakeLists.txt
@@ -10,7 +10,7 @@ INSTALL(CODE "EXECUTE_PROCESS(COMMAND ${CMAKE_COMMAND} -DTARGET=${PYTHON_SITE_PA
 
 if (NOT PYKDEUIC4_ALTINSTALL)
     # Step 2: Make a the generated symlink a link to "pykdeuic4"
-    INSTALL(CODE "EXECUTE_PROCESS(COMMAND ${CMAKE_COMMAND} -DTARGET=${BIN_INSTALL_DIR}/${_uic_name} -DLINK_NAME=${BIN_INSTALL_DIR}/pykdeuic4 -P ${cmake_module_dir}/create_exe_symlink.cmake)" )
+    install(CODE "EXECUTE_PROCESS(COMMAND ${CMAKE_COMMAND} -E create_symlink ${BIN_INSTALL_DIR}/${_uic_name} ${BIN_INSTALL_DIR}/pykdeuic4)" )
 endif()
 
 unset(_uic_name)
-- 
1.8.1

